app/src/main/java/woowacourse/movie/movie/MainActivity.kt

```diff
@@ -0,0 +1,88 @@
+package woowacourse.movie.movie
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Bundle
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.commit
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import woowacourse.movie.R
+import woowacourse.movie.databinding.ActivityMainBinding
+import woowacourse.movie.movie.history.HistoryFragment
+import woowacourse.movie.movie.movielist.HomeFragment
+import woowacourse.movie.movie.setting.SettingFragment
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        requestNotificationPermission()
+        binding.navigationBar?.let { bindNavigationBar(it) }
```

Comment:
```suggestion
        binding.navigationBar?.let(::bindNavigationBar)
```
다음과 같이 함수 레퍼런스를 이용해 작성할 수도 있답니다!